cmsimple/classes/LinkChecker.php

```diff
@@ -277,7 +334,7 @@ public function reportError(Link $link)
         global $tx;
 
         $o = '<li>' . "\n" . '<b>' . $tx['link']['link'] . '</b>'
-            . '<a href="' . $link->getURL() . '">' . $link->getText() . '</a>'
+            . '<a target="_blank" href="' . $link->getURL() . '">' . $link->getText() . '</a>'
```

Comment:
Ich bin noch immer kein Freund von target=_blank, aber da könnte ich mich auch bei den Browserherstellern beschweren; auf jeden Fall nichts, worüber ich streiten möchte. Kann von mir aus also so bleiben.

Reply:
Das ist sicher Ansichtssache und auch immer wieder Streitpunkt.
Aber in dem Fall ist es auf jeden Fall besser.
Wenn man den Link manuell prüfen will und nicht daran denkt einen neunen Tab aufzumachen, muss man bzw. wird anschließend die gesamte Prüfung wiederholt.

Reply:
Ah, stimmt! Langfristig sollten wir da insgesamt nachbessern, aber für jetzt ist das eine ordentliche Lösung. Danke!